src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1441,9 +1441,6 @@ I agree to all of the foregoing:</value>
   <data name="Prefer_simplified_boolean_expressions" xml:space="preserve">
     <value>Prefer simplified boolean expressions</value>
   </data>
-  <data name="Razor_CSharp_Language_Server_Client" xml:space="preserve">
```

Comment:
I think the all the translations should be removed as well

Reply:
Where do you all store the translations?

Reply:
Here - https://github.com/dotnet/roslyn/tree/master/src/VisualStudio/Core/Def/xlf - all the xlf with the same name (ServicesVSResources)

I think a commandline build should fix them?  If not there was some msbuild flag to pass to regenerate

Reply:
I think /t:UpdateXlf